tests/fv3_conf/cpld_fv3_mom6_cice_ww3_bmark_rt_run.IN

```diff
@@ -44,12 +48,44 @@ cp    @[RTPWD]/CICE_FIX/@[OCNRES]/mesh.mx@[OCNRES].nc .
 cp    @[RTPWD]/WW3_input_data/mod_def.* .
 cp    @[RTPWD]/WW3_input_data/ww3_multi.inp .
 
-# ICs
-cp    @[RTPWD]/${FV3_IC}/sfc_data*.nc ./INPUT
-cp    @[RTPWD]/${FV3_IC}/gfs_data*.nc ./INPUT
-cp    @[RTPWD]/${MOM6_IC}/MOM*.nc ./INPUT
+# No restart
+if [ $WARM_START = .F. ]; then
+  # ICs
+  if [ $FRAC_GRID = .F. ]; then
+  # cp  @[RTPWD]/${FV3_DIR}/INPUT/sfc_data*.nc ./INPUT
+  # cp  @[RTPWD]/${FV3_DIR}/INPUT/gfs_data*.nc ./INPUT
+   cp  @[RTPWD]/${FV3_IC}/sfc_data*.nc ./INPUT
+   cp  @[RTPWD]/${FV3_IC}/gfs_data*.nc ./INPUT
+  else
+   cp  @[RTPWD]/FV3_input_frac/@[ATMRES].mx@[OCNRES]_frac/sfc_data*.nc ./INPUT
```

Comment:
I don't think this is correct. The bmark RT runs on the date 2013-04-01. Doesn't the FV3_input_frac directory contain the ICs for the "normal" RT day of 2016-10-03?

I think other additional test (atm_flux_frac) is OK since that is running on the date of 2016-10-03.